robottelo/constants/__init__.py

```diff
@@ -6,9 +6,9 @@
 from nailgun import entities
 
 # This should be updated after each version branch
-SATELLITE_VERSION = "6.17"
+SATELLITE_VERSION = "6.16"
 SATELLITE_OS_VERSION = "8"
-SAT_NON_GA_VERSIONS = ['6.17']
+SAT_NON_GA_VERSIONS = ['6.16', '6.17']
```

Comment:
Need to asses the impact having two versions here as if they are being looped to operate on both versions.

Reply:
@jameerpathan111 What about `['6.16']` to skip the need for an assessment  ?

Reply:
@jyejare SAT_NON_GA_VERSIONS is used in only repositories_setup tests, which just validates the repositories usage for upgrades, and this tests are passing in PRT

Reply:
You can check `6.13.z`, `6.14.z`, `6.15.z`, `6.16.z` branches that we have everything unreleased as nobody goes back to N-2 branch to adjust this constant.
E.g you can see in `6.13.z` branch:
```
SAT_NON_GA_VERSIONS = ['6.14', '6.15']
```
But that is very much useless as automation in 6.12.z branch does care only about `6.12` in SAT_NON_GA_VERSIONS  - this would be better just boolean flag or yet better...


This shall be config value.

Reply:
I've assessed it and the only operation is `not in SAT_NON_GA_VERSIONS` so it is going to be OK with multiple values.   (there is no operation like `SAT_NON_GA_VERSIONS[0]`)